packages/api/ecs/async-operation/README.md

```diff
@@ -2,15 +2,15 @@
 
 This image is intended for use with the AsyncOperation model. It will download
 an AWS Lambda function from AWS, download a payload from S3, execute the lambda
-function (inside the Docker container), then update the DynamoDB record for the
+function (inside the Docker container), then update the RDS database record for the
```

Comment:
It technically still updates both

Reply:
I'd rather not call it out in every single place, because I think it's defeating the point of the doc update which is what people should be seeing RDS as the source of truth. I think we should swap the docs over for the most part to be RDS.